src/EFCore.Jet/Query/Sql/Internal/IJetExpressionVisitor.cs

```diff
@@ -1,13 +1,9 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.Linq.Expressions;
-using EntityFrameworkCore.Jet.Query.Expressions.Internal;
-using JetBrains.Annotations;
-
 namespace EntityFrameworkCore.Jet.Query.Sql.Internal
 {
     public interface IJetExpressionVisitor
```

Comment:
In theory it could be. But I kept `RowNumber` commented-out in the code for future reference as an example how to implement functionality in similar fashion throughout the code.